hnn_core/cell.py

```diff
@@ -220,6 +224,52 @@ def __init__(self, L, diam, Ra, cm, end_pts=None):
     def __repr__(self):
         return f'L={self.L}, diam={self.diam}, cm={self.cm}, Ra={self.Ra}'
 
+    def __eq__(self, other):
```

Comment:
Are there two `__eq__` functions for this class? I think one is being overridden

Reply:
There is one each for Section and Cell classes